src/SDKs/Batch/Management/Management.Batch.Tests/Management.Batch.Tests.csproj

```diff
@@ -12,7 +12,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="[1.1.3-preview,2.0.0)" />
-    <ProjectReference Include="..\Management.Batch\Microsoft.Azure.Management.Batch.csproj" />
+    <ProjectReference Include="..\Management.Batch\Microsoft.Azure.Management.Batch.csproj" />    
```

Comment:
Frank I see this file is still being changed.
Can you remove this unwanted file change from your commit?

Reply:
This was a mistakenly touched in my branch and there is no changes in this file.